rlel/Account.xaml.cs

```diff
@@ -163,5 +191,17 @@ public partial class Account : UserControl {
         private void UserControl_MouseDoubleClick(object sender, System.Windows.Input.MouseButtonEventArgs e) {
             this.launchAccount();
         }
+
+        public bool ClientIsRunning()
+        {
+            if (this.main.singularity.IsChecked == true && this.tranqProcess != null)
+                return !this.tranqProcess.HasExited;
+            else if (this.main.singularity.IsChecked == false && this.sisiProcess != null)
+                return !this.tranqProcess.HasExited;
```

Comment:
Since the function is not used yet I didn't notice but your right there was a mixup.
And `this.main.singularity.IsChecked` is of type `bool?` and not `bool` so the simple way wont work. And its also checked in the rest of the code with `==`